kytos.json

```diff
@@ -0,0 +1,11 @@
+{
+  "author": "kytos",
+  "username": "kytos",
+  "name": "of_multi_table",
+  "description": "This NApp implements Oplenflow multi tables",
+  "version": "1.0",
+  "napp_dependencies": [],
```

Comment:
Could you include `"kytos/flow_manager"` in `napp_dependencies` for information and explicitness? right now this property isn't being strictly used but we want to keep it updated. 

The other NApps that `of_multi_table` will subscribe to their events I'm still not sure if we should mark as dependency since it's like a dynamic dependency. We also have other NApps like `mef_eline` subscribing to `topology`'s event but it's not marked as a dependency, so maybe should be fine leaving them out, but then in the README event docs as they start to get documented we included there, OK?

Reply:
Added